library/std/src/sys/windows/process.rs

```diff
@@ -133,6 +190,13 @@ impl Command {
     pub fn creation_flags(&mut self, flags: u32) {
         self.flags = flags;
     }
+    #[allow(dead_code)]
```

Comment:
What is this for?

Reply:
There seem to be a lot of these, some pre-existing 🤷